### PR TITLE
feat(protocol-designer): add base router url for sandbox deploys

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

This PR adds a base router url for sandbox deploys. This is so that we preserve the full url route in sandbox builds. Prior to this PR, if you hit `sandbox.designer.opentrons.com/${BRANCH_NAME}`, react router dom would redirect you to `sandbox.designer.opentrons.com/${ROUTE}` instead of `sandbox.designer.opentrons.com/${BRANCH_NAME}/${ROUTE}`

## Changelog

- add base name to router for sandbox artifacts

## Review requests

- make sure sandbox routes are preserved in PD at https://sandbox.designer.opentrons.com/pd_add-base-route-to-router/

## Risk assessment

Medium, but also this isn't in prod yet

### CODE REVIEW

File: protocol-designer/src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -26,9 +26,8 @@ export function Landing(): JSX.Element {
 
   React.useEffect(() => {
     if (metadata?.created != null) {
+      console.warn('protocol already exists, navigating to overview')
```
Comment (shlokamin) [inline protocol-designer/src/pages/Landing/index.tsx:29]: i added this console warn cuz debugging react router dom side effects is suuuuper painful. when possible/appropriate i think its a good idea to add a log message when an unexpected redirect needs to happen

File: protocol-designer/src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -26,9 +26,8 @@ export function Landing(): JSX.Element {
...
       navigate('/overview')
-    } else {
-      navigate('/')
     }
```
Comment (shlokamin) [inline protocol-designer/src/pages/Landing/index.tsx:31]: this component is already on the landing page route, i don't think this is necessary

File: .github/workflows/pd-test-build-deploy.yaml [hunk 3]
```diff
@@ -162,7 +162,7 @@ jobs:
           OT_PD_MIXPANEL_ID: ${{ secrets.OT_PD_MIXPANEL_ID }}
           OT_PD_MIXPANEL_DEV_ID: ${{ secrets.OT_PD_MIXPANEL_DEV_ID }}
         run: |
-          make -C protocol-designer
+          make -C protocol-designer NODE_ENV=development
```
Comment (shlokamin) [inline .github/workflows/pd-test-build-deploy.yaml:165]: sandbox builds were set to run in prod mode, which i don't think we want

File: .github/workflows/pd-test-build-deploy.yaml [hunk 4]
```diff
@@ -215,4 +215,5 @@ jobs:
           aws configure set role_arn ${{ secrets.OT_PD_DEPLOY_ROLE }} --profile deploy
           aws configure set source_profile identity --profile deploy
           aws s3 sync ./dist s3://sandbox.designer.opentrons.com/${{ env.OT_BRANCH }} --acl=public-read --profile=deploy
+          aws cloudfront create-invalidation --distribution-id ${{ secrets.PD_CLOUDFRONT_SANDBOX_DISTRIBUTION_ID }} --paths "/*" --profile deploy
```
Comment (shlokamin) [inline .github/workflows/pd-test-build-deploy.yaml:219]: PD sandbox deploys weren't updating consistently on push so i added an explicit cloudfront invalidation on sandbox pushes... @y3rsh FYI

**Review by jerader — APPROVED** [review]
thanks Shlok! seems to work as expected

**Pull request merged**